src/components/ActivityCollector/attachClickActivityCollector.js

```diff
@@ -17,6 +17,7 @@ const createClickHandler = ({ eventManager, lifecycle, handleError }) => {
     // TODO: Consider safeguarding from the same object being clicked multiple times in rapid succession?
     const clickedElement = clickEvent.target;
     const event = eventManager.createEvent();
+    event.documentMayUnload();
```

Comment:
Can we add an inline comment explaining why we have added this expression?

Reply:
sure!